gateway-ha/src/main/java/io/trino/gateway/ha/handler/DropWizardProxyHandlerStats.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.handler;
+
+import com.codahale.metrics.Meter;
+import io.airlift.stats.CounterStat;
+import io.dropwizard.core.setup.Environment;
```

Comment:
lets not add new Dropwizard dependencies in new PRs

Reply:
Sorry - I should have explained that this is a refactor of existing code to allow for testing. 

This class was `ProxyHandlerStats`, which was a pain to use in unit tests because of the Dropwizard setup. So I made `ProxyHandlerStats` into an interface, made this class an implementation of it, and added the `TestingProxyHandlerStats` implementation.